taperole/hosts

```diff
@@ -1,6 +1,6 @@
 # Base Example
 [production]
-162.243.113.173 be_app_env=production be_app_branch=deployment
+162.243.113.173 be_app_env=production be_app_branch=bugfix/api-keys
```

Comment:
sounds good!  thanks for the explanation

On Mon, Feb 15, 2016 at 2:53 PM, Tyler Rockwell <notifications@github.com>
wrote:

> In taperole/hosts
> <https://github.com/TylerRockwell/bookstore/pull/32#discussion_r52937926>:
>
> > @@ -1,6 +1,6 @@
> >  # Base Example
> >  [production]
> > -162.243.113.173 be_app_env=production be_app_branch=deployment
> > +162.243.113.173 be_app_env=production be_app_branch=bugfix/api-keys
>
> I was working on a different branch and needed to make sure the things I
> changed were working. I'll change it to master when everything's merged.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/TylerRockwell/bookstore/pull/32/files#r52937926>.
>